assets/javascripts/discourse/services/post-event-api.js

```diff
@@ -0,0 +1,30 @@
+import Service from "@ember/service";
+import { ajax } from "discourse/lib/ajax";
+
+/**
+ * Discourse Post Event API Service
+ *
+ * @implements {@ember/service}
+ */
+export default class PostEventApi extends Service {
```

Comment:
I'm not sure this should be a service... it feels like it would be noisy if ajax calls were are following the same pattern. Not sure services are an appropriate vehicle for this, I think it's fine in this case to have the ajax call in the component.

Reply:
That's a good question, I'm not sure we currently have an official pattern. Some core API calls are in the model, for example the `user` model has a list. 

Might be worth a longer internal discussion.